cdci_data_analysis/analysis/queries.py

```diff
@@ -500,6 +500,8 @@ def get_query_products(self,instrument,job,run_asynch,query_type='Real',logger=N
 
                 if job_status=='done':
                     job.set_done()
+                elif job_status == 'failed':
+                    job.set_failed()
```

Comment:
forgot to ask, is this ok to be done here ?

Reply:
I guess so.